modules/client-configuration/pages/registration-overview-suse.adoc

```diff
@@ -19,9 +19,36 @@ You must also have created an activation key.
 For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
 
 
+
 [IMPORTANT]
 ====
 Do not register a {productname} Server to itself.
 The {productname} Server must be managed individually or by using another separate {productname} Server.
 For more information about using multiple servers, see xref:specialized-guides:large-deployments/hub-multi-server.adoc[].
 ====
+
+
+
+[[suse-registration-overview-check]]
+== Check Previous Registration Status
+
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
+If you forget about removing the old registration first, it can happen that repositories not needed anymore getting automatically added to the client even after the client has been migrated to a newer version of a {suse} operating system using {productname}.
+
+* To de-register a {sle} 11 client, on the client, as root, execute:
++
+
+----
+suse_register -E
+----
```

Comment:
I think we do not need SLES11. It is not supported in SUMA (officially) so we should not document anything about it